src/js/panels.js

```diff
@@ -1053,4 +1053,45 @@ https://github.com/fluid-project/first-discovery/raw/master/LICENSE.txt
         }
     });
 
+    /*
+     * Token panel
+     */
+    fluid.defaults("gpii.firstDiscovery.panel.token", {
+        gradeNames: ["fluid.prefs.panel"],
+        preferenceMap: {
+            "gpii.firstDiscovery.token": {}
+        },
+        selectors: {
+            message: ".gpiic-fd-token-message",
+            token: ".gpiic-fd-token"
+        },
+        selectorsToIgnore: ["token"],
```

Comment:
I started with having token as a model value but realized it's more straightforward using events since the token (or the error) is only updated once at the panel instantiation. The token is not intended to be changed once it's set. Firing an event when the server response is ready and parsed seems making more sense.